src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -317,7 +260,12 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
           <Spacer y={4} />
           <Flex width="100%" justifyContent="space-between">
             <Button
-              onClick={handleBackToEdit}
+              onClick={() => {
+                handleCloseModal({
```

Comment:
If I understand correctly from the [spreadsheet](https://docs.google.com/spreadsheets/d/1c65vugkF2C4YuXwBXno8aqsBh4B2fTwxiLmgY_MhjWc/edit#gid=0), we want to track both the `ClickedValidationAddressOptions` and `ClickedCloseValidationAddressModal` event here.

Reply:
Confirmed behavior with dayane and updated per her request!